README.md

```diff
@@ -297,6 +297,8 @@ Unlock the vehicle.
 | Type               | Description         |
 |:------------------ |:--------------------|
 | Dictionary.`status`| Set to success on successful request. |
+#### Throws
+<code>SmartcarError</code> on unsuccessful request
```

Comment:
called `SmartcarException` in Python.

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html it's also `raise` in python. Take a look at this link for the standards - https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.

you may want to add the `raise` above the code block too!